packages/runtime/datastore/src/channelStorageService.ts

```diff
@@ -25,7 +26,7 @@ export class ChannelStorageService implements IChannelStorageService {
 
     constructor(
         private readonly tree: ISnapshotTree | undefined,
-        private readonly storage: Pick<IDocumentStorageService, "read">,
+        private readonly storage: Pick<IDocumentStorageService, "readBlob">,
         private readonly extraBlobs?: Map<string, string>,
```

Comment:
nit: This should become Map<string, ArrayBufferLike> and the callers should be updated to pass ArrayBufferLike for the blobs.